### PR TITLE
feat: ui mdm/oda enrollment remediation

### PR DESCRIPTION
**Pull request by ivandenysenko-okta**
ui view implementation

OKTA-340805

## Description:

Screen Shot 2020-11-11 at 6 24 59 PM
Screen Shot 2020-11-11 at 6 25 55 PM

## PR Checklist

- [x] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Issue:

- [OKTA-340805](https://oktainc.atlassian.net/browse/OKTA-340805)

### CODE REVIEW

File: src/v2/view-builder/components/OdaOktaVerifyTerminalView.js [hunk 1]
```diff
@@ -25,6 +28,7 @@ export default View.extend({
     const templateData = {
       signInUrl: deviceEnrollment.signInUrl,
       appStoreLogoClass: `${platform}-app-store-logo`,
+      copy: loc(`enroll.copy.${platform}`, 'login'),
```
Comment (haishengwu-okta) [inline src/v2/view-builder/components/OdaOktaVerifyTerminalView.js:31]: nit:
seems you can use `i18n` in the template

Comment (haishengwu-okta) [reply src/v2/view-builder/components/OdaOktaVerifyTerminalView.js:31]: nvm, missed that `platform` is variable here.

**Review by stevennguyen-okta — COMMENTED** [review]
Is there a secondary text color available other than dark-text-color? I would like to de-emphasize the legal copy so it is not on the same level as the instructional text

**Comment by ivandenysenko-okta** [comment]
@stevennguyen-okta there is `$light-text-color: #a7a7a7;`
Screen Shot 2020-11-10 at 7 26 12 PM

**Comment by stevennguyen-okta** [comment]
> @stevennguyen-okta there is `$light-text-color: #a7a7a7;`
> Screen Shot 2020-11-10 at 7 26 12 PM

Is that the

> @stevennguyen-okta there is `$light-text-color: #a7a7a7;`
> Screen Shot 2020-11-10 at 7 26 12 PM

Per the slack thread https://okta.slack.com/archives/CAZH2MWEL/p1605031326298000 , we can proceed with Odyssey's Gray-600 color for the legal footer

**Comment by ivandenysenko-okta** [comment]
@stevennguyen-okta updated screenshots
odyssey gray-600 is #6e6e78

File: assets/sass/v2/_device-enrollment-terminal.scss [hunk 2]
```diff
@@ -41,4 +41,10 @@
       background-image: url('../img/appstore/apple-app-store.svg');
     }
   }
+  .copy {
+    font-size: 10px;
+    font-style: italic;
+    color: #6e6e78;
```
Comment (haishengwu-okta) [inline assets/sass/v2/_device-enrollment-terminal.scss:47]: add comments saying this is `odyssey cv('gray', '600');`

**Review by haishengwu-okta — APPROVED** [review]

**Review by stevennguyen-okta — APPROVED** [review]
Looks good, thanks for the changes!

**Pull request closed without merging**